desc/objectives/getters.py

```diff
@@ -172,19 +176,107 @@ def get_NAE_constraints(
 
+    constraints += (
+        FixNearAxisR(
+            eq=desc_eq,
+            target=qsc_eq,
+            N=N,
+            order=order,
+            normalize=normalize,
+        ),
+        FixNearAxisZ(
+            eq=desc_eq,
+            target=qsc_eq,
+            N=N,
+            order=order,
+            normalize=normalize,
+        ),
+    )
+    if fix_lambda or (fix_lambda >= 0 and type(fix_lambda) is int):
+        constraints += (
+            FixNearAxisLambda(
+                eq=desc_eq,
+                target=qsc_eq,
+                N=N,
+                order=int(fix_lambda),
+                normalize=normalize,
+            ),
+        )
+
+    return constraints
+
+
+def _get_NAE_constraints(
+    desc_eq,
+    qsc_eq,
+    order=1,
+    N=None,
+    fix_lambda=False,
+    normalize=True,
+):
+    """Get the constraints necessary for fixing NAE behavior in an equilibrium problem.
+
+    NOTE: This will return tuples of FixSumModes__, this is not intended to be directly
+    used by the user. Instead, call the ``get_NAE_constraints`` function, or use the
+    FixNearAxis{R,Z,Lambda} objectives along with the FixAxis{R,Z} objectives. This
+    instead is a helper function to get the needed constraints for the
+
+    Parameters
+    ----------
+    desc_eq : Equilibrium
+        Equilibrium to constrain behavior of
+        (assumed to be a fit from the NAE equil using `.from_near_axis()`).
+    qsc_eq : Qsc
+        Qsc object defining the near-axis equilibrium to constrain behavior to.
+        order : {0,1,2}
+        order (in rho) of near-axis behavior to constrain
+    normalize : bool
+        Whether to apply constraints in normalized units.
+    N : int
+        max toroidal resolution to constrain.
+        If `None`, defaults to equilibrium's toroidal resolution
+    fix_lambda : bool or int
+        Whether to constrain lambda to match that of the NAE near-axis
+        if an `int`, fixes lambda up to that order in rho {0,1}
+        if `True`, fixes lambda up to the specified order given by `order`
+    normalize : bool
+        Whether to apply constraints in normalized units.
+
+    Returns
+    -------
+    constraints, tuple of _Objectives
+        A list of the linear constraints used in fixed-near-axis behavior problems.
+    """
+    if not isinstance(fix_lambda, bool):
+        fix_lambda = int(fix_lambda)
+    constraints = ()
+
+    kwargs = {"eq": desc_eq, "normalize": normalize, "normalize_target": normalize}
+    if not isinstance(fix_lambda, bool):
+        fix_lambda = int(fix_lambda)
+
     if fix_lambda or (fix_lambda >= 0 and type(fix_lambda) is int):
         L_axis_constraints, _, _ = calc_zeroth_order_lambda(
             qsc=qsc_eq, desc_eq=desc_eq, N=N
         )
         constraints += L_axis_constraints
+
+    # Axis constraints
+    constraints += (
+        FixAxisR(**kwargs),
+        FixAxisZ(**kwargs),
+        AxisRSelfConsistency(desc_eq),
```

Comment:
Yes, but in order for the `FixNearAxis` objectives to also fix the axis, they require the `A` matrix relating `R_lmn` and `Ra_n` which is actually created inside the self consistency objectives. So I include this here so that the `FixNearAxis` objective can have all the info it needs to fix the axis